src/main/java/org/skriptlang/skript/lang/structure/Structure.java

```diff
@@ -169,6 +171,16 @@ public Priority getPriority() {
 		return DEFAULT_PRIORITY;
 	}
 
+	@Override
+	public final List<Class<? extends Section>> getUsableSections() {
+		return SyntaxElement.super.getUsableSections();
+	}
+
+	@Override
+	public final List<Class<? extends Structure>> getUsableStructures() {
+		return SyntaxElement.super.getUsableStructures();
```

Comment:
```suggestion
		return super.getUsableStructures();
```

Reply:
this doesn't work, i have to specify SyntaxElement